packages/grafana-data/src/utils/logs.test.ts

```diff
@@ -70,7 +70,7 @@ describe('calculateLogsLabelStats()', () => {
       },
     ];
 
-    expect(calculateLogsLabelStats(rows as any, 'baz')).toEqual([]);
+    expect(calculateLogsLabelStats(rows as unknown as LogRowModel[], 'baz')).toEqual([]);
```

Comment:
I would suggest to fix this directly where you create `rows` variable rather than where you use it. So something like:
```
 const rows = [
      {
        entry: 'foo 1',
        labels: {
          foo: 'bar',
        },
      },
    ] as unknown as LogRowModel[];
```

This way, if you would use `rows` multiple times, you don't need to specify it for each function call. It also keeps type close to variable. This would apply to other assertions as well.

Reply:
Or even better - figuring out why we have to use `unknown`. Looking into this, it is because in `LogRowModel`, `labels` is supposed to be type of `Labels`, but in our test it is just an object. And when we look at `Labels` type, it is expected to be object `interface Labels { [key: string]: string;}`. So what we can do is: 

```
    const rows = [
      {
        entry: 'foo 1',
        labels: {
          foo: 'bar',
        } as Labels,
      },
    ] as LogRowModel[];
    ```